tools/cpplint.py

```diff
@@ -3021,7 +3021,7 @@ def CheckComment(line, filename, linenum, next_line_start, error):
       # If the comment contains an alphanumeric character, there
       # should be a space somewhere between it and the // unless
       # it's a /// or //! Doxygen comment.
-      if (Match(r'//[^ ]*\w', comment) and
+      if (Match(r'(?://[^ ]*\w)|(?:////\s*$)', comment) and
```

Comment:
Do we maintain our own `cpplint` or is it updated from upstream from time to time?

Reply:
Good question, I don't know the answer I'm afraid. Hopefully someone can chime in. I'd be happy to open a PR upstream if this is the case.

Reply:
Seems like we maintain our own judging by the file's history.

Reply:
LGTM then.